content/javascript/concepts/date-methods/getDay.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: ".getDay()"
+Subjects:
+  - "Web Development"
+  - "Computer Science"
+Tags:
+  - "Date"
+  - "Methods"
+Catalog Content:
+  - "https://www.codecademy.com/learn/paths/web-development"
+---
+
+## Definition
+
+Get the day of the week for the specified date according to local time.
+
+## Syntax
+
+```js
+Date.getDay();
+```
+
+The return value of `getDay()` will only be an integer number between `0` and `6`.
```

Comment:
Combine the below line with this one. 

The return value of `.getDay()` will be an integer number between `0` and `6`, with `0` meaning Sunday, `1` meaning Monday, and so on.